history.js

```diff
@@ -413,7 +422,7 @@
    */
   function parseURL(href, isWindowLocation, isNotAPI) {
     var re = /(?:([a-zA-Z0-9\-]+\:))?(?:\/\/(?:[^@]*@)?([^\/:\?#]+)(?::([0-9]+))?)?([^\?#]*)(?:(\?[^#]+)|\?)?(?:(#.*))?/;
-    if (href !== null && href !== '' && !isWindowLocation) {
+    if (isPresent(href) && href !== '' && !isWindowLocation) {
```

Comment:
`href != null` is the same as `isPresent(href)` so I do not see any reason to create an extra function.